GeeksCoreLibrary/Components/Configurator/Services/ConfiguratorsService.cs

```diff
@@ -459,17 +459,22 @@ public async Task<VueConfiguratorDataModel> GetVueConfiguratorDataAsync(string n
         validationRegexErrorMessage.`value` AS validationRegexErrorMessage,
         IF(
             templatesFromStepId.id IS NOT NULL AND templatesFromStepId.`value` NOT IN ('', '0'),
-            (SELECT CONCAT_WS('', `value`, long_value) FROM wiser_itemdetail WHERE item_id = templatesFromStepId.`value` AND `key` = 'step_template'),
+            (SELECT CONCAT_WS('', `value`, long_value) FROM wiser_itemdetail WHERE item_id = CONVERT(templatesFromStepId.`value`, UNSIGNED) AND `key` = 'step_template'),
             CONCAT_WS('', stepTemplate.`value`, stepTemplate.long_value)
         ) AS stepTemplate,
         IF(
             templatesFromStepId.id IS NOT NULL AND templatesFromStepId.`value` NOT IN ('', '0'),
-            (SELECT CONCAT_WS('', `value`, long_value) FROM wiser_itemdetail WHERE item_id = templatesFromStepId.`value` AND `key` = 'values_template'),
+            (SELECT CONCAT_WS('', `value`, long_value) FROM wiser_itemdetail WHERE item_id = CONVERT(templatesFromStepId.`value`, UNSIGNED) AND `key` = 'values_template'),
```

Comment:
Please use the constants in WiserTableNames.

Reply:
Fixed.